fe/fe-core/src/main/java/org/apache/doris/catalog/BrokerMgr.java

```diff
@@ -306,6 +306,22 @@ public void replayDropAllBroker(String name) {
         }
     }
 
+    public void replayUpdateBrokerInfo(String name, List<FsBroker> brokers) {
+        lock.lock();
+        try {
+            for (FsBroker oldBroker : brokerListMap.get(name)) {
+                for (FsBroker newBroker : brokers) {
+                    if (oldBroker.hostName != null && oldBroker.hostName.equals(newBroker.hostName)
+                        && oldBroker.port == newBroker.port) {
+                        oldBroker.ip = newBroker.ip;
```

Comment:
something is wrong here. brokermap has ip-->broker. If changed ip here, should also change the map.